pages/docs/guides/validate.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Validate AsyncAPI Documents"
+description: In this guide, we'll learn multiple ways to validate AsyncAPI documents.
+weight: 120
+---
+
+# Introduction
+In this guide, we'll learn multiple ways to validate AsyncAPI documents. 
+
+When people talk about AsyncAPI validation, sometimes they mean completely different things: **validation of AsyncAPI documents** or **validation of messages in runtime** _(message schemas provided in the AsyncAPI document)_.  
+
+# Prequisites
+Because this guide covers multiple options for validating AsyncAPI documents, there are different tool options you might choose to install:
+- [AsyncAPI CLI](https://github.com/asyncapi/cli#installation)
+- [Spectral](https://meta.stoplight.io/docs/spectral/b8391e051b7d8-installation)
+
+# Create AsyncAPI documents
```

Comment:
I don't think you need to explain what asyncapi is.

Turn it into a section that says: "yo, to validate AsyncAPi document, you first need to have one, right 😉 if you did not write one for your app yet, you can easily generate one using one of our examples provided in the AsyncAPI CLI" -> and then line 20-24 is good